src/atomate2/common/jobs/pheasy.py

```diff
@@ -0,0 +1,475 @@
+    ther_cond_temp: list[int],
+    sym_reduce: bool,
+    symprec: float,
+    use_symmetrized_structure: str | None,
+    kpath_scheme: str,
+    code: str,
+    mp_id: str,
+    displacement_data: dict[str, list],
+    total_dft_energy: float,
+    epsilon_static: Matrix3D = None,
+    born: Matrix3D = None,
+    **kwargs,
+) -> PhononBSDOSDoc:
+    """
+    Analyze the phonon runs and summarize the results.
+
+    Parameters
+    ----------
+    structure: Structure object
+        Fully optimized structure used for phonon runs
+    supercell_matrix: np.array
+        array to describe supercell
+    displacement: float
+        displacement in Angstrom used for supercell computation
+    sym_reduce: bool
+        if True, symmetry will be used in phonopy
+    symprec: float
+        precision to determine symmetry
+    use_symmetrized_structure: str
+        primitive, conventional, None are allowed
+    kpath_scheme: str
+        kpath scheme for phonon band structure computation
+    code: str
+        code to run computations
+    displacement_data: dict
+        outputs from displacements
+    total_dft_energy: float
+        total DFT energy in eV per cell
+    epsilon_static: Matrix3D
+        The high-frequency dielectric constant
+    born: Matrix3D
+        Born charges
+    kwargs: dict
+        Additional parameters that are passed to PhononBSDOSDoc.from_forces_born
+    """
+    return PhononBSDOSDoc.from_forces_born(
+        structure=structure.remove_site_property(property_name="magmom")
+        if "magmom" in structure.site_properties
+        else structure,
+        supercell_matrix=supercell_matrix,
+        displacement=displacement,
+        num_displaced_supercells=num_displaced_supercells,
+        cal_anhar_fcs=cal_anhar_fcs,
+        displacement_anhar=displacement_anhar,
+        num_disp_anhar=num_disp_anhar,
+        fcs_cutoff_radius=fcs_cutoff_radius,
+        renorm_phonon=renorm_phonon,
+        renorm_temp=renorm_temp,
+        cal_ther_cond=cal_ther_cond,
+        ther_cond_mesh=ther_cond_mesh,
+        ther_cond_temp=ther_cond_temp,
+        sym_reduce=sym_reduce,
+        symprec=symprec,
+        use_symmetrized_structure=use_symmetrized_structure,
+        kpath_scheme=kpath_scheme,
+        code=code,
+        mp_id=mp_id,
+        displacement_data=displacement_data,
+        total_dft_energy=total_dft_energy,
+        epsilon_static=epsilon_static,
+        born=born,
+        **kwargs,
+    )
+
+
+# I did not directly import this job from the phonon module
+# because I modified the job to pass the displaced structures
+# to the output.
+@job(data=["forces", "displaced_structures"])
+def run_phonon_displacements(
+    displacements: list[Structure],
+    structure: Structure,
+    supercell_matrix: Matrix3D,
+    phonon_maker: BaseVaspMaker | ForceFieldStaticMaker | BaseAimsMaker = None,
+    prev_dir: str | Path = None,
+    prev_dir_argname: str = None,
+    socket: bool = False,
+) -> Flow:
```

Comment:
if possible, i would modify the phonons.py job instead to avoid code duplication